docs/docsite/rst/scenario_guides/guide_online.rst

```diff
@@ -0,0 +1,29 @@
+*************************
+Using Ansible with Online
+*************************
+
+Introduction
+============
+
+Online is a French hosting company mainly known for providing baremetal servers named Dedibox.
+Check it out: https://www.online.net/en
+
+Dynamic inventory for Online resources
+--------------------------------------
+
+Ansible has a dynamic inventory plugin that can list your resources.
+
+1. Create a YAML configuration such as ``online_inventory.yml``
+
+.. code-block:: yaml
+
+    plugin: online
+
+2. Set your ``ONLINE_TOKEN`` environment variable with your token.
+   You can find your token at the following page: https://console.online.net/en/api/access
+
+3. You can test that your inventory is working by running: ``ansible-inventory -v -i online_inventory.yml --list``
+
+4. Now you can run your playbook or any other module with this inventory: ``ansible all -i online_inventory.yml -m ping``
+
+If python is installed on your machine, you should have a pong answer to this command.
```

Comment:
Python is required to run Ansible, so I think you should remove the first phrase. It would be nice to add sample output (what the user would see) here.